test/terraform_aws_nextjs_serverless_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"crypto/tls"
+	"fmt"
+	"strings"
+	"testing"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/tidwall/gjson"
+)
+
+func TestTerraformCloudFrontServerlessNextJS(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: "../examples/nextjs-v13/terraform/",
+		Reconfigure:  true,
+		// The path of the var file in relation to TerraformDir
+		VarFiles: []string{"../../../config/terratest.tfvars"},
+	})
+
+	// At the end of the test, run `terraform destroy` to clean up any resources that were created
```

Comment:
I don't think those comments are necessary. You usally just comment code in case it's unclear what it does.